config.yaml

```diff
@@ -1,24 +1,24 @@
 fields:
   - name: test
     nullRate: 0.1
-    generators: [ net.datafaker.providers.base.Name#lastName]
```

Comment:
it would make sense to keep at least one with abs name since it's also a case that should keep working

Reply:
Done.